### PR TITLE
Paulo Barata

### PR DESCRIPTION
**Pull request by PBara7a**

### CODE REVIEW

File: src/cohort.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    return student
+  }
+
+  removeStudent(id) {
+    const studentToRemove = this.students.findIndex(
+      (student) => student.id === id
+    )
+    if (studentToRemove === -1) throw new Error('Student not found')
+
+    return this.students.splice(studentToRemove, 1)[0]
```
Comment (Anicaaa) [inline src/cohort.js:22]: I'd like to know more about `.splice()`. I have used it a few times before, but you have an index `[0]` next to the splice method. I wonder what it does.

File: src/manager.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+  getCohort(cohortName) {
+    const cohort = this.cohorts.find((cohort) => cohort.name === cohortName)
+    if (cohort) return cohort
+    return false
+  }
+
+  createCohort(cohortName) {
+    if (!cohortName) return 'Cannot create a cohort without a name.'
+    const cohortExists = this.cohorts.find((el) => el.name === cohortName)
```
Comment (Anicaaa) [inline src/manager.js]: I wonder what that `(el)` next to the `.find()` method does.

**Comment by Anicaaa** [comment]
Your code is concise and not overwhelming to read at all!
I'm hoping to code like you one day. Good job 👏🏽

File: src/cohort.js (hunk 1 above)
Comment (PBara7a) [reply src/cohort.js:22]: .splice() returns an array containing the deleted elements. Here I am removing a single student and I want to return that student Without the index [0] the method would return an array containing the student, but accessing the index [0] makes it return the student at index 0 of the array created by splice().

File: src/manager.js (hunk 2 above)
Comment (PBara7a) [reply src/manager.js]: el (short for element) represents the current element in the cohorts array. For each el, it checks if the name of el is equal to cohortName (the name passed into the method to create a cohort.

**Pull request closed without merging**